src/www/index.html

```diff
@@ -3,7 +3,7 @@
 
 <head>
   <title>WireGuard</title>
-  <link href="/css/vendor/tailwind.min.css" rel="stylesheet">
+  <link href="css/vendor/tailwind.min.css" rel="stylesheet">
```

Comment:
I prefer `./` to indicate it's a relative path.

Reply:
I can make the change. It worked, so I did it.